src/components/header.js

```diff
@@ -29,6 +30,7 @@ class Header extends React.Component<Props, State> {
   }
   render() {
     const { path } = this.props
+    let blogPath = "https://blog.digitallibrary.io/"
```

Comment:
Tanken er at man skal kunne nå `(test|staging|demo).blog.digitallibrary.io` når man er i `test|staging|demo` også. Med denne så når man alltid produksjon uansett hvilket miljø man er i :)

For å vite hvilket miljø man kjører i, så bruker vi noe som heter environment variabler. Så hvis du prøver å kjøre opp appen med `GATSBY_GDL_ENVIRONMENT=test` så skal du i appen kunne si `process.env.GATSBY_GDL_ENVIRONMENT` for å få ut "test", og dermed kunne sette riktig url.